.changeset/itchy-kiwis-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
```

Comment:
Probably want to add a little more information here as to what people have to do with their current installation with a like a diff so that people can migrate easier.

Reply:
I wrote some more info here. Let me know what you think.